spec/models/office_spec.rb

```diff
@@ -20,6 +20,35 @@
     it 'managers' do
       expect(office).to respond_to(:managers)
     end
+
+    it 'jurisdictions' do
+      expect(office).to respond_to(:jurisdictions)
+    end
+  end
+
+  describe 'jurisdiction' do
+    it 'can be nil' do
```

Comment:
Maybe here there can be:

```ruby
before(:each) { office.jurisdictions.clear }
```

instead of repeating 

```ruby
office.jurisdictions.clear
```